agent/util-scripts/pbench-tool-data-sink

```diff
@@ -105,12 +106,14 @@ class ToolDataSink(Bottle):
     class Terminate(Exception):
         pass
 
-    def __init__(self, redis_server, channel, benchmark_run_dir, logger):
+    def __init__(self, redis_server, channel, benchmark_run_dir, list_hosts, list_tools, logger):
```

Comment:
Seems like this work is not layered on top of #8.

I'll incorporate #8 into Tool Meister work first, and then we can have you rebase on top of it.